shared/src/store/action-reducers/simulated-region.ts

```diff
@@ -111,6 +111,18 @@ export class AddBehaviorToSimulatedRegionAction implements Action {
     public readonly behaviorState!: ExerciseSimulationBehaviorState;
 }
 
+export class RemoveBehaviorFromSimulatedRegionAction implements Action {
+    @IsValue('[SimulatedRegion] Remove Behavior' as const)
+    public readonly type = '[SimulatedRegion] Remove Behavior';
+
+    @IsUUID(4, uuidValidationOptions)
+    public readonly simulatedRegionId!: UUID;
+
+    @Type(...simulationBehaviorTypeOptions)
+    @ValidateNested()
+    public readonly behaviorState!: ExerciseSimulationBehaviorState;
```

Comment:
I would rather use the behavior ID only to keep the action small.

Reply:
done